src/Picqer/Financials/Exact/BankEntry.php

```diff
@@ -33,8 +33,6 @@ class BankEntry extends Model
 
     protected $primaryKey = 'EntryID';
 
-    protected $bankEntryLines = [];
```

Comment:
This should not be deleted. Perhaps for all collections properties such an internal property is needed although this property is unused (never initialised, never accessed).